src/smefit/fit_manager.py

```diff
@@ -149,6 +155,33 @@ def smeft_predictions(self):
             )
         return np.array(smeft)
 
+    @property
+    def smeft_predictions_best_fit(self):
+        """Compute |SMEFT| predictions for the best fit point.
+
+        Returns
+        -------
+        np.ndarray:
+            |SMEFT| predictions for the best fit
+        """
+
+        best_fit_point = np.array(
```

Comment:
The best_fit_point array should also contain the constrained coefficients, currently it doesn't

Reply:
What do you mean, sorry?

Reply:
At the moment it doesn't contain the values of the constrained coefficients, such as Cll1111, and so the SMEFT predictions you compute on line 175 do not take into account its effect.

Reply:
But it does the same of the function smeft_predictions, are they both bugged?

Reply:
No, I just checked and there the constraints are propagated, i.e. cll1111 is included and is set to cll. I can have a look where the difference arises.